### ISSUE TITLE
Jetson nano 2 test failed 

### ISSUE THREAD

**Issue by linux-devil** [RidgeRun/gst-interpipe#48]
I tried make check on jetson nano , 2 test failed 
```
PASS: gst/test_allow_renegotiation_property
PASS: gst/test_anonymous_connection
PASS: gst/test_block_switch
PASS: gst/test_buffer_properties
PASS: gst/test_caps_renegotiation
FAIL: gst/test_enable_sync
PASS: gst/test_get_caps
PASS: gst/test_hot_plug
PASS: gst/test_in_bounds_events
FAIL: gst/test_interpipe_synchronization
PASS: gst/test_invalid_caps
PASS: gst/test_node_name_removed
PASS: gst/test_out_of_bounds_events
PASS: gst/test_out_of_bounds_upstream_events
PASS: gst/test_reconfigure_event
PASS: gst/test_set_caps
============================================================================
Testsuite summary for GstInterpipe 1.1.0
============================================================================
# TOTAL: 16
# PASS:  14
# SKIP:  0
# XFAIL: 0
# FAIL:  2
# XPASS: 0
# ERROR: 0

```

**Comment by Santiago-vdk** [comment]
Same error. Could the plugin still function without these tests passing?

**Comment by jcaballeros** [comment]
Duplicate of #49

**Comment by jcaballeros** [comment]
Hi,

This issue has been resolved in v1.1.2.
